crates/fluvio/src/producer/error.rs

```diff
@@ -9,8 +9,8 @@ use crate::producer::PartitionId;
 #[derive(thiserror::Error, Debug, Clone)]
 #[non_exhaustive]
 pub enum ProducerError {
-    #[error("the given record is larger than the max_request_size ({0} bytes)")]
-    RecordTooLarge(usize),
+    #[error("record size ({0} bytes), exceeded maximum request size ({0} bytes)")]
```

Comment:
since api is changed, need to bump major version

Reply:
sure! I bumped the fluvio major version